app/views/devise/passwords/edit.html.erb

```diff
@@ -12,12 +12,12 @@
     ) do |f| %>
   <%= f.input :reset_password_token, as: :hidden %>
   <%= f.full_error :reset_password_token %>
-  <%= render PasswordToggleComponent.new(
+  <%= render PasswordConfirmationComponent.new(
         form: f,
+        password_label: t('forms.passwords.edit.labels.password'),
         field_options: {
-          label: t('forms.passwords.edit.labels.password'),
-          required: true,
           input_html: {
+            aria: { describedby: 'password-description' },
```

Comment:
I like the idea of associating with the preceding description like we do elsewhere, but the paragraph on this page doesn't have the ID, so this is invalid.

Should we add the ID to the paragraph?

Reply:
Added, but would we want to also add the mirrored content:

```
<p id="password-description">
  <%= t('instructions.password.info.lead_html', min_length: Devise.password_length.first) %>
</p>
````

Reply:
I don't necessarily think we need to change the content, though you could check with the UX folks on the team if you think it'd be good to bring the content in sync.